numpy/lib/twodim_base.py

```diff
@@ -136,12 +204,11 @@ def flipud(m):
     return m[::-1, ...]
 
 
-def rot90(m, k=1):
+def rot90(m, k=1, axes=(0,1)):
     """
-    Rotate an array by 90 degrees in the counter-clockwise direction.
+    Rotate an array by 90 degrees counter-clockwise in the plane specified by axes.
```

Comment:
Based on @shoyer's comment, counter clockwise should be omitted. It is true for the original function since it was equivalent to `rot90(m, k, (0, 1))`, but it's no longer true since you can do `rot90(m, k, (1, 0))`, which is in the same plane but clockwise.